build.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+# Open Chakra Toning
+# Copyright (C) 2024  Nicco Kunzmann
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#
+# Build the flatpack.
+#
+
+builddir=".build"
+id="io.github.niccokunzmann.python_dhcp_server"
+manifest="$id.yml"
+repo=".repo"
+reponame="python_dhcp_server_repo"
+builder="flatpak run org.flatpak.Builder"
+SUDO=""
+
+set -e
+cd "`dirname \"$0\"`"
+
+echo "Install the build environment"
+## see https://docs.flatpak.org/en/latest/first-build.html
+if ! flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo; then
+    sudo flatpak remote-add --if-not-exists flathub https://dl.flathub.org/repo/flathub.flatpakrepo
+    SUDO=sudo
+else
+    USER="--user"
+fi
```

Comment:
I'll repeat one more time. YOU SHOULD NEVER RUN FLATPAK AS ROOT.

Reply:
Yes, I heard you!